packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_class_table.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../../../primitives/utils.dart';
+import '../../../../shared/table.dart';
+import '../../../../shared/table_data.dart';
+import '../../../../shared/utils.dart';
+import 'allocation_profile_tracing_view_controller.dart';
+
+/// The default width for columns containing *mostly* numeric data (e.g.,
+/// instances, memory).
+const _defaultNumberFieldWidth = 80.0;
+
+class _TraceCheckBox extends ColumnData<TracedClass>
+    implements ColumnRenderer<TracedClass> {
+  _TraceCheckBox({
+    required this.controller,
+    required this.setState,
+  }) : super(
+          'Trace',
+          titleTooltip:
+              'Enable or disable allocation tracing for a specific type',
+          fixedWidthPx: scaleByFontFactor(55.0),
+          alignment: ColumnAlignment.left,
+        );
+
+  final AllocationProfileTracingViewController controller;
+
+  /// The `setState` function for [AllocationTracingTable], used to trigger a
+  /// rebuild of the table when allocation tracing is enabled or disabled as a
+  /// result of a checkbox state change in this column.
+  final Function(Function()) setState;
```

Comment:
Not sure if this is a bit smelly, but it seemed like this simplest way to force the table to rebuild.